api/libproject/devstart.rb

```diff
@@ -1929,19 +1925,19 @@ def list_runtimes(cmd_name, *args)
   op = WbOptionsParser.new(cmd_name, args)
   gcc = GcloudContextV2.new(op)
   op.add_option(
-      "--runtime-project [project]",
```

Comment:
"runtime project" is ambiguous: do you mean the billing project (AKA workspace namespace) or the google project?

I think think is a leftover from the time when these values were equal.  today they are not.